utils/benchmark/benchmarkTest.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Add customized environment configuration
+mv server/config/environment.js server/config/environment.orig.js
+cp "$ROOT_DIR_IMA/utils/benchmark/app/environment.js" server/config/environment.js
+NODE_ENV=prod node server/server.js &
+IMA_SKELETON_SERVER_PID=$!
```

Comment:
We are saving the server PID here, but we never do the cleanup part, so the server runs until the end of CI job. We should stop this process after running the autocannon test.

Reply:
Oh, I see that you are sourcing this file in the `setupForTests.sh`, so it actually gets cleared right now after all the tests. But still, since starting the server is part of this benchmark test, then we should also clear it as part of this test, so we have all the logic around this test in one place.

Reply:
Command `kill $IMA_SKELETON_SERVER_PID` moved to `createImaAppTests.sh`